src/Traits/QueueWithoutOverlap.php

```diff
@@ -31,12 +32,14 @@ public function queue($queue, $command)
     /**
      * Lock this job's key in redis, so no other
      * jobs can run with the same key.
+     * @param Lock|null $lock
      * @return bool - false if it fails to lock
      */
-    protected function acquireLock(array $lock = [])
+    protected function acquireLock(Lock $lock = null)
     {
         $lock_time = isset($this->lock_time) ? $this->lock_time : 300; // in seconds; 5 minutes default
```

Comment:
This could use `??` to simplify it a little (since the line below does), and I'm a little surprised 300 isn't a class constant like `RedLock::DEFAULT_LOCK_TIME`